app/assets/stylesheets/_colors.scss

```diff
@@ -0,0 +1,29 @@
+$smoke: #08090a;
+$smoke-90: #202428;
```

Comment:
@ludwiczakpawel are the -x the lightness or darkness increments?

Reply:
```
$smoke-90 --> dark
$smoke-00 --> light
```

`$smoke` is just a default base color, which gonna be the darkest. it could actually also have an alias `$smoke-100`.

Also, keep in mind these are not imported anywhere, so they are not ready to use. I'm just preparing our codebase for implementing design system stuff..